android-exoplayer/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -131,29 +129,25 @@ private void updateShutterViewVisibility() {
     }
 
     /**
-     * Set the {@link SimpleExoPlayer} to use. The {@link SimpleExoPlayer#addTextOutput} and
-     * {@link SimpleExoPlayer#addVideoListener} method of the player will be called and previous
+     * Set the {@link ExoPlayer} to use. The {@link ExoPlayer#addListener} method of the
+     * player will be called and previous
      * assignments are overridden.
      *
-     * @param player The {@link SimpleExoPlayer} to use.
+     * @param player The {@link ExoPlayer} to use.
      */
-    public void setPlayer(SimpleExoPlayer player) {
+    public void setPlayer(ExoPlayer player) {
         if (this.player == player) {
             return;
         }
         if (this.player != null) {
-            this.player.removeTextOutput(componentListener);
-            this.player.removeVideoListener(componentListener);
```

Comment:
What happened where we no longer need these listeners?

Reply:
`VideoListener` and `TextOutput` are deprecated and their methods was merged to `Player.Listener` instead. For `TextOutput` it is `onCues` method and for `VideoListener` it is `VideoListener`